---
title: 'About'
layout: 'page'
---


# ISARIC4C Consortium leadership

- Kenneth Baillie (j.k.baillie@ed.ac.uk)
- Calum Semple (M.G.Semple@liverpool.ac.uk)
- Peter Openshaw (p.openshaw@imperial.ac.uk)

# Management

- Recruitment, sample tracking and ethical approvals: Hayley Hardwick (H.E.Jelleyman@liverpool.ac.uk)
- MTAs, IDAMAC and Consortium Management: Wilna Oosthuyzen(Wilna.Oosthuyzen@ed.ac.uk)

# Objectives and Resources

*Each Work Package has a lead and deputy who will be responsible for resource allocation and leadership*

*Each deliverable has a single consortium member who will be responsible for ensuring delivery*

**WP1. Coordination** (*Baillie, Semple, Openshaw(co-leads), Russell, Ho, Docherty, Noursadeghi, Law*)

- Coordinate timely delivery across consortium(*Baillie*).[WEEKS 0-78]
    - DELIVERABLE: Extend existing collaboration agreement. Baillie.[WEEKS 0-2]
    - DELIVERABLE: Coordinate rapid delivery across all WPs. Baillie.[WEEKS 0-78]
    - DELIVERABLE: Database for sharing results. Law.[WEEKS 0-10]
    - DELIVERABLE: Browser interface for sharing results. Docherty.[WEEKS 4-20]
    - DELIVERABLE: Completion of proposed analyses in public domain. Baillie.[WEEKS 0-78]
    - DELIVERABLE: Anonymise clinical data to prevent personal disclosures. Doherty.[WEEKS 0-1]
    - DELIVERABLE: Submission to WHO sharepoint. Law.[WEEKS 0-78]
- Lead Recruitment of 1300 confirmed cases (*Semple*).[WEEKS 0-78]
    - DELIVERABLE: Recruit 300 cases at TIER2(4 serial samples). Semple.[WEEKS 0-40]
    - DELIVERABLE: Recruit 1000 cases at TIER1(1 sample). Semple.[WEEKS 30-60]
    - DELIVERABLE: Provide additional research nurse support flexibly to sites. Semple.[WEEKS 3-55]
    - DELIVERABLE: Establish and review 4-month contracts with sites for reserach nurse support. Semple.[WEEKS 3-55]
    - DELIVERABLE: Establish distrubution of samples to consortium and partners. Semple.[WEEKS 0-78]
    - DELIVERABLE: Recruit and distribute samples in Scotland. Ho.[WEEKS 0-40]
- Lead Recruitment 1000 suspected cases(*Noursadeghi*).[WEEKS 0-52]
    - DELIVERABLE: Establish BioAid clone as substudy of ISARIC CCP. Noursadeghi.[WEEKS 0-4]
    - DELIVERABLE: Recruit 1000 suspected cases at presentation. Noursadeghi.[WEEKS 0-52]

**WP2. Viral transmission** (*Barclay(lead), Dunning(deputy), Zambon, Ho, Hiscox*)

- Idenfity sites and duration of viral shedding(*Hiscox*)[WEEKS 0-40]
    - DELIVERABLE: PCR on SAM strips,  Urine + stool. Hiscox.[WEEKS 0-40]
- Quantify Aerosol spread in hospitalised cases(*Barclay*)[WEEKS 0-15]
    - DELIVERABLE: Aerosol transmission study. Barclay.[WEEKS 0-15]
    - DELIVERABLE: Quantify the effect of high-flow nasal oxyen on viral aerosolisation. Barclay.[WEEKS 0-15]

**WP3. Pathogen characteristics and evolution** (*Hiscox(lead), Thomson(deputy), Zambon, Rambaut, Bogaert*)

- Identify viral sequence correlates with severe disease(*Hiscox, Thomson, Zambon*)[WEEKS 0-40]
    - DELIVERABLE: Deep MinION sequencing for metagenomics on 80 respiratory (SAM) samples. Hiscox/Thomson.[WEEKS 0-8]
    - DELIVERABLE: Amplicon MinION sequencing for metagenomics on 300 respiratory (SAM) samples. Hiscox/Thomson.[WEEKS 4-40]
    - DELIVERABLE: Analysis of viral selection during infection. Thomson.[WEEKS 12-45]
- Map and track viral transmission and evolution(with PHE).(*Rambaut*)[WEEKS 0-78]
    - DELIVERABLE: Analysis of viral transmission and evolution. Rambaut.[WEEKS 0-78]
- Detect bacterial/viral/fungal coinfection(*Hiscox, Bogaert*)[WEEKS 0-40]
    - DELIVERABLE: Microbiome sequencing in respiratory samples. Bogaert/Hiscox.[WEEKS 10-40]

**WP4. Clinical characterisation** (*Semple(lead), Baillie(deputy), Openshaw, Summers, Docherty, Russell, Dunning, Ho, Cooke*)

- Define dynamics of viral production disease.(*Docherty*)[WEEKS 0-40]
    - DELIVERABLE: Collate,  integrate and interpret viral shedding data. Docherty.[WEEKS 0-40]
    - DELIVERABLE: Synthesise viral load data in clinical context. Semple.[WEEKS 0-40]
- Create prognostic scores(*Ho, Noursadeghi*)[WEEKS 0-24]
    - DELIVERABLE: Clincal prognostic score. Ho.[WEEKS 0-24]
    - DELIVERABLE: Molecular prognostic score. Noursadeghi.[WEEKS 20-48]
- Determine role of nonpulmonary organ injury(*Docherty*)[WEEKS 0-24]
    - DELIVERABLE: Clinical lab tests on serial samples (U&E,  LFT(AST),  HSTrop,  Ferritin,  Trigs,  CRP,  Procalcitonin,  D-dimer,  LDH,  BNP,  CK,  Amylase). Baillie.[WEEKS 0-40]
    - DELIVERABLE: Quantify clinical severity for every case + timepoint. Doherty.[WEEKS 6-12]
    - DELIVERABLE: Analyse and report clinical description. Doherty.[WEEKS 6-12]

**WP5. Integrative analysis of disease mechanisms** (*Baillie(lead), Meynert(deputy), Thomson, Hiscox, Openshaw, Russell, Klenerman, Sancho Shimizu*)

- RNAseq quantitation of systemic viral RNAemia and host transcriptome.(*Thomson/Meynert*)[WEEKS 0-40]
    - DELIVERABLE: Whole blood RNA extraction and storage. Baillie.[WEEKS 0-60]
    - DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 8-10]
    - DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 18-20]
    - DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 28-30]
    - DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 38-40]
    - DELIVERABLE: Align viral reads from whole blood RNAseq. Thomson.[WEEKS 4-40]
    - DELIVERABLE: Upload viral reads to ENA. Meynert.[WEEKS 12-40]
    - DELIVERABLE: Upload host reads to ENA. Meynert.[WEEKS 12-40]
    - DELIVERABLE: Align and quantify viral reads. Meynert.[WEEKS 12-40]
    - DELIVERABLE: Align and quantify host reads. Meynert.[WEEKS 12-40]
- Integrate consortium data to estimate *tolerance* and *resistance*.(*Russell*)[WEEKS 0-40]
    - DELIVERABLE: Define integrated tolerance and resistance patterns. Russell.[WEEKS 4-40]
    - DELIVERABLE: Differential expression analysis. Parkinson.[WEEKS 11-42]
- Identify patient subgroups/trajectories.(*Baillie*)[WEEKS 0-60]
    - DELIVERABLE: Unsupervised clustering of disease trajectories. Baillie.[WEEKS 0-40]
- Detect genetic associations with severe disease.(*Meynert*)[WEEKS 0-60]
    - DELIVERABLE: Cell pellet DNA extraction and storage. Baillie.[WEEKS 0-60]
    - DELIVERABLE: Genome-wide array genotyping. Baillie.[WEEKS 15-18]
    - DELIVERABLE: Genome-wide array genotyping. Baillie.[WEEKS 35-38]
    - DELIVERABLE: GWAS analyses. Meynert.[WEEKS 20-44]
    - DELIVERABLE: Link with known eQTL. Meynert.[WEEKS 0-40]
    - DELIVERABLE: WGS. Baillie.(Â£606 - 27% unit cost for seq)[WEEKS 20-30]
    - DELIVERABLE: Identify high-consequence variants in WGS. Sancho Shimizu.[WEEKS 30-45]

**WP6. Immunology** (*Openshaw(lead), Klenerman(deputy), Thwaites, Zambon, Ijaz, Thomson, Turtle, Dunning, Solomon, Stuart*)

- Describe and characterise the host immune response.(*Openshaw*)[WEEKS 12-22]
    - DELIVERABLE: Cytokine and proteomic analyses in respiratory specimens. Openshaw.[WEEKS 12-30]
    - DELIVERABLE: Cytokine analyses in blood. Openshaw.[WEEKS 12-30]
    - DELIVERABLE: Proteomic analyses in 60 blood samples. Semple.[WEEKS 12-22]
- Determine effects of prior immunity to coronaviruses on susceptibility.(*Thomson, Zambon, Ijaz*)[WEEKS 0-40]
    - DELIVERABLE: Serological tests for prior exposure to coronaviruses. Ijaz.[WEEKS 6-18]
    - DELIVERABLE: NGS Serological tests for prior exposure to all resp viruses. Thomson.[WEEKS 12-22]
    - DELIVERABLE: Integrate clinical,  transcriptomic,  NGS serology and neutralisation assays to determine ADE effect. Ballie.[WEEKS 30-40]
- Describe B&T cell responses in a subset of patients(*Klenerman*)[WEEKS 4-16]
    - DELIVERABLE: Obtain PBMCs from a subset of acute cases. Turtle.[WEEKS 4-16]
    - DELIVERABLE: T-cell functional assays. Klenerman.[WEEKS 4-16]
- Identify antigens targeted by neutralising antibodies.(*Stuart*)[WEEKS 0-26]
    - DELIVERABLE: Determine structure of antibody-binding sites. Stuart.[WEEKS 0-40]
- Identify nasal cell populations supporting viral replication(*Thwaites*)[WEEKS 8-14]
    - DELIVERABLE: Single-cell (nucelar) sequencing on 20 nasopharyngeal (SAM) samples. Thwaites.[WEEKS 8-14]
- Identify circulating immune cell subpopulations associated with severe disease.(*Klenerman*)[WEEKS 0-20]
    - DELIVERABLE: Single-cell (nuclear) sequencing on 60 selected frozen cell pellet samples. Klenerman.[WEEKS 8-14]

**WP7. Diagnostics and immunotherapeutics** (*Mentzer, Ijaz, Zambon(co-leads), Pollakis, Noursadeghi, Klenerman*)

- Obtain large-volume convalescent samples from 8 patients.(*Mentzer*)[WEEKS 0-10]
    - DELIVERABLE: Select 8 patients and arrange transfer to Oxford. Mentzer.[WEEKS 0-4]
    - DELIVERABLE: Obtain 8x200mls convalescent plasma and PBMCs. Mentzer.[WEEKS 0-4]
- Distribute serum and PBMCs to PHE and external groups(*Semple, Mentzer*)[WEEKS 0-60]
    - DELIVERABLE: Cyropreserve and distribute convalescent PMBC as instructed by IDAMAC. Mentzer.[WEEKS 0-10]
    - DELIVERABLE: Store and distribute convalescent plasma as instructed by IDAMAC. Mentzer.[WEEKS 0-10]
    - DELIVERABLE: Provide samples to define international reference standard. Mentzer.[WEEKS 0-1]
- Establish open-access platform for large-scale diagnostic evaluation.(*Mentzer*)[WEEKS 0-52]
    - DELIVERABLE: Platform for comparisons of diagnostics. Mentzer.[WEEKS 8-65]
    - DELIVERABLE: Validate oral fluid test for population serology. Tedder.[WEEKS 20-32]
- CL2 SARS-CoV-2 virus neutralization assay(*Pollakis/Paxton*)[WEEKS 0-52]
    - DELIVERABLE: Produce retroviral pseudo-virus,  optimize for infectivity. Pollakis.[WEEKS 0-6]
    - DELIVERABLE: Determine best target cell for infection and sensitive readout. Pollakis.[WEEKS 4-8]
    - DELIVERABLE: Selection of high neutralizing sera for relative quantification of responses. Paxton.[WEEKS 8-20]
    - DELIVERABLE: Develop SARS-CoV-2-specific neutralization protocol. Paxton.[WEEKS 8-20]
    - DELIVERABLE: Convalescent sera neutralizing response measurements and cross-validation with serology assays. Paxton/Pollakis.[WEEKS 20-52]
    - DELIVERABLE: Upscaling of neutralization assay for high throughput. Pollakis.[WEEKS 20-34]
    - DELIVERABLE: High throughput neutralization measurement on all samples. Pollakis.[WEEKS 35-48]

**WP8. Facilitating translation to local clinical guidelines** (*Alex(lead), Bach(deputy), Grover, Baillie, Russell*)

- AI-driven guideline/protocol sharing system for local clinicians.(*Alex, Bach*)[WEEKS 0-30]
    - DELIVERABLE: Collate, compare and present local clinical guidelines. Russell.[WEEKS 0-10]
    - DELIVERABLE: Create AI engine. Alex.[WEEKS 0-30].
    - DELIVERABLE: Create web interface. Bach.[WEEKS 0-20].

**WP9. Long-term follow-up (unfunded)** (*Lone(lead), Semple(deputy), Jenkins, Turtle, Docherty, Baillie, Russell, many others...*)
- Serial serology (Semple, Turtle)
    - DELIVERABLE: Serial serology and immunology follow-up of selected cases. Semple.
- Clincal follow-up (Lone, Jenkins)
    - DELIVERABLE: Data linkage follow-up of all cases. Lone.
    - DELIVERABLE: Clinical follow-up of selected cases. Jenkins.








