<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link href="https://fonts.googleapis.com/css?family=Comfortaa&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet">
<!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Consortium leadership | isaric4c.github.io</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Consortium leadership" />
<meta property="og:locale" content="en_US" />
<link rel="canonical" href="http://localhost:4000/about/structure.html" />
<meta property="og:url" content="http://localhost:4000/about/structure.html" />
<meta property="og:site_name" content="isaric4c.github.io" />
<script type="application/ld+json">
{"headline":"Consortium leadership","@type":"WebPage","url":"http://localhost:4000/about/structure.html","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->

    <link rel="stylesheet" href="/assets/css/style.css?v=4efd7f2e2337cf885e464e59ac8d67792d99c122">
    <!--[if lt IE 9]>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
    <![endif]-->
    <link rel="apple-touch-icon" sizes="180x180" href="/img/favicon/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/img/favicon/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/img/favicon/favicon-16x16.png">
    <link rel="manifest" href="/img/favicon/site.webmanifest">
    <link rel="mask-icon" href="/img/favicon/safari-pinned-tab.svg" color="#5bbad5">
    <link rel="shortcut icon" href="/img/favicon/favicon.ico">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="msapplication-config" content="/img/favicon/browserconfig.xml">
    <meta name="theme-color" content="#ffffff">
  </head>
  <body>
    <div class="wrapper">
      <header>
        <table>
           <tr>
             <td>
              <a href="/index.html">Home</a>
              <br/>
              <a href="/info">Results to date - All Open Access</a>
              <br/>
              <a href="/risk">Risk calculator</a>
              <br/>
              <a href="/sample_access">Data and sample access requests, protocols and CRF</a>
              <br/>
              <a href="/setup">Site set up</a>
              <br/>
              <a href="/privacy">Privacy Notice</a>
              <br/>
              <td>
                <a href="/"><img width="100px" src="/img/isaric4c-logo.png" alt="Logo"/></a>
             </td>
          </tr>
        </table>
     </header>

       <section>

      <h1 id="consortium-leadership">Consortium leadership</h1>

<ul>
  <li>Kenneth Baillie (j.k.baillie@ed.ac.uk)</li>
  <li>Calum Semple (M.G.Semple@liverpool.ac.uk)</li>
  <li>Peter Openshaw (p.openshaw@imperial.ac.uk)</li>
</ul>

<h1 id="management">Management</h1>

<ul>
  <li>Recruitment, sample tracking and ethical approvals: Hayley Hardwick (H.E.Jelleyman@liverpool.ac.uk)</li>
  <li>MTAs, IDAMAC and Consortium Management: Jane Ewins(jane.ewins@ed.ac.uk) and Wilna Oosthuyzen(Wilna.Oosthuyzen@ed.ac.uk)</li>
</ul>

<h1 id="objectives-and-resources">Objectives and Resources</h1>

<p><em>Each Work Package has a lead and deputy who will be responsible for resource allocation and leadership</em></p>

<p><em>Each deliverable has a single consortium member who will be responsible for ensuring delivery</em></p>

<p><strong>WP1. Coordination</strong> (<em>Baillie, Semple, Openshaw(co-leads), Russell, Ho, Docherty, Noursadeghi, Law</em>)</p>

<ul>
  <li>Coordinate timely delivery across consortium(<em>Baillie</em>).[WEEKS 0-78]
    <ul>
      <li>DELIVERABLE: Extend existing collaboration agreement. Baillie.[WEEKS 0-2]</li>
      <li>DELIVERABLE: Coordinate rapid delivery across all WPs. Baillie.[WEEKS 0-78]</li>
      <li>DELIVERABLE: Database for sharing results. Law.[WEEKS 0-10]</li>
      <li>DELIVERABLE: Browser interface for sharing results. Docherty.[WEEKS 4-20]</li>
      <li>DELIVERABLE: Completion of proposed analyses in public domain. Baillie.[WEEKS 0-78]</li>
      <li>DELIVERABLE: Anonymise clinical data to prevent personal disclosures. Doherty.[WEEKS 0-1]</li>
      <li>DELIVERABLE: Submission to WHO sharepoint. Law.[WEEKS 0-78]</li>
    </ul>
  </li>
  <li>Lead Recruitment of 1300 confirmed cases (<em>Semple</em>).[WEEKS 0-78]
    <ul>
      <li>DELIVERABLE: Recruit 300 cases at TIER2(4 serial samples). Semple.[WEEKS 0-40]</li>
      <li>DELIVERABLE: Recruit 1000 cases at TIER1(1 sample). Semple.[WEEKS 30-60]</li>
      <li>DELIVERABLE: Provide additional research nurse support flexibly to sites. Semple.[WEEKS 3-55]</li>
      <li>DELIVERABLE: Establish and review 4-month contracts with sites for reserach nurse support. Semple.[WEEKS 3-55]</li>
      <li>DELIVERABLE: Establish distrubution of samples to consortium and partners. Semple.[WEEKS 0-78]</li>
      <li>DELIVERABLE: Recruit and distribute samples in Scotland. Ho.[WEEKS 0-40]</li>
    </ul>
  </li>
  <li>Lead Recruitment 1000 suspected cases(<em>Noursadeghi</em>).[WEEKS 0-52]
    <ul>
      <li>DELIVERABLE: Establish BioAid clone as substudy of ISARIC CCP. Noursadeghi.[WEEKS 0-4]</li>
      <li>DELIVERABLE: Recruit 1000 suspected cases at presentation. Noursadeghi.[WEEKS 0-52]</li>
    </ul>
  </li>
</ul>

<p><strong>WP2. Viral transmission</strong> (<em>Barclay(lead), Dunning(deputy), Zambon, Ho, Hiscox</em>)</p>

<ul>
  <li>Idenfity sites and duration of viral shedding(<em>Hiscox</em>)[WEEKS 0-40]
    <ul>
      <li>DELIVERABLE: PCR on SAM strips,  Urine + stool. Hiscox.[WEEKS 0-40]</li>
    </ul>
  </li>
  <li>Quantify Aerosol spread in hospitalised cases(<em>Barclay</em>)[WEEKS 0-15]
    <ul>
      <li>DELIVERABLE: Aerosol transmission study. Barclay.[WEEKS 0-15]</li>
      <li>DELIVERABLE: Quantify the effect of high-flow nasal oxyen on viral aerosolisation. Barclay.[WEEKS 0-15]</li>
    </ul>
  </li>
</ul>

<p><strong>WP3. Pathogen characteristics and evolution</strong> (<em>Hiscox(lead), Thomson(deputy), Zambon, Rambaut, Bogaert</em>)</p>

<ul>
  <li>Identify viral sequence correlates with severe disease(<em>Hiscox, Thomson, Zambon</em>)[WEEKS 0-40]
    <ul>
      <li>DELIVERABLE: Deep MinION sequencing for metagenomics on 80 respiratory (SAM) samples. Hiscox/Thomson.[WEEKS 0-8]</li>
      <li>DELIVERABLE: Amplicon MinION sequencing for metagenomics on 300 respiratory (SAM) samples. Hiscox/Thomson.[WEEKS 4-40]</li>
      <li>DELIVERABLE: Analysis of viral selection during infection. Thomson.[WEEKS 12-45]</li>
    </ul>
  </li>
  <li>Map and track viral transmission and evolution(with PHE).(<em>Rambaut</em>)[WEEKS 0-78]
    <ul>
      <li>DELIVERABLE: Analysis of viral transmission and evolution. Rambaut.[WEEKS 0-78]</li>
    </ul>
  </li>
  <li>Detect bacterial/viral/fungal coinfection(<em>Hiscox, Bogaert</em>)[WEEKS 0-40]
    <ul>
      <li>DELIVERABLE: Microbiome sequencing in respiratory samples. Bogaert/Hiscox.[WEEKS 10-40]</li>
    </ul>
  </li>
</ul>

<p><strong>WP4. Clinical characterisation</strong> (<em>Semple(lead), Baillie(deputy), Openshaw, Summers, Docherty, Russell, Dunning, Ho, Cooke</em>)</p>

<ul>
  <li>Define dynamics of viral production disease.(<em>Docherty</em>)[WEEKS 0-40]
    <ul>
      <li>DELIVERABLE: Collate,  integrate and interpret viral shedding data. Docherty.[WEEKS 0-40]</li>
      <li>DELIVERABLE: Synthesise viral load data in clinical context. Semple.[WEEKS 0-40]</li>
    </ul>
  </li>
  <li>Create prognostic scores(<em>Ho, Noursadeghi</em>)[WEEKS 0-24]
    <ul>
      <li>DELIVERABLE: Clincal prognostic score. Ho.[WEEKS 0-24]</li>
      <li>DELIVERABLE: Molecular prognostic score. Noursadeghi.[WEEKS 20-48]</li>
    </ul>
  </li>
  <li>Determine role of nonpulmonary organ injury(<em>Docherty</em>)[WEEKS 0-24]
    <ul>
      <li>DELIVERABLE: Clinical lab tests on serial samples (U&amp;E,  LFT(AST),  HSTrop,  Ferritin,  Trigs,  CRP,  Procalcitonin,  D-dimer,  LDH,  BNP,  CK,  Amylase). Baillie.[WEEKS 0-40]</li>
      <li>DELIVERABLE: Quantify clinical severity for every case + timepoint. Doherty.[WEEKS 6-12]</li>
      <li>DELIVERABLE: Analyse and report clinical description. Doherty.[WEEKS 6-12]</li>
    </ul>
  </li>
</ul>

<p><strong>WP5. Integrative analysis of disease mechanisms</strong> (<em>Baillie(lead), Meynert(deputy), Thomson, Hiscox, Openshaw, Russell, Klenerman, Sancho Shimizu</em>)</p>

<ul>
  <li>RNAseq quantitation of systemic viral RNAemia and host transcriptome.(<em>Thomson/Meynert</em>)[WEEKS 0-40]
    <ul>
      <li>DELIVERABLE: Whole blood RNA extraction and storage. Baillie.[WEEKS 0-60]</li>
      <li>DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 8-10]</li>
      <li>DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 18-20]</li>
      <li>DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 28-30]</li>
      <li>DELIVERABLE: Host/viral RNAseq. Baillie.[WEEKS 38-40]</li>
      <li>DELIVERABLE: Align viral reads from whole blood RNAseq. Thomson.[WEEKS 4-40]</li>
      <li>DELIVERABLE: Upload viral reads to ENA. Meynert.[WEEKS 12-40]</li>
      <li>DELIVERABLE: Upload host reads to ENA. Meynert.[WEEKS 12-40]</li>
      <li>DELIVERABLE: Align and quantify viral reads. Meynert.[WEEKS 12-40]</li>
      <li>DELIVERABLE: Align and quantify host reads. Meynert.[WEEKS 12-40]</li>
    </ul>
  </li>
  <li>Integrate consortium data to estimate <em>tolerance</em> and <em>resistance</em>.(<em>Russell</em>)[WEEKS 0-40]
    <ul>
      <li>DELIVERABLE: Define integrated tolerance and resistance patterns. Russell.[WEEKS 4-40]</li>
      <li>DELIVERABLE: Differential expression analysis. Parkinson.[WEEKS 11-42]</li>
    </ul>
  </li>
  <li>Identify patient subgroups/trajectories.(<em>Baillie</em>)[WEEKS 0-60]
    <ul>
      <li>DELIVERABLE: Unsupervised clustering of disease trajectories. Baillie.[WEEKS 0-40]</li>
    </ul>
  </li>
  <li>Detect genetic associations with severe disease.(<em>Meynert</em>)[WEEKS 0-60]
    <ul>
      <li>DELIVERABLE: Cell pellet DNA extraction and storage. Baillie.[WEEKS 0-60]</li>
      <li>DELIVERABLE: Genome-wide array genotyping. Baillie.[WEEKS 15-18]</li>
      <li>DELIVERABLE: Genome-wide array genotyping. Baillie.[WEEKS 35-38]</li>
      <li>DELIVERABLE: GWAS analyses. Meynert.[WEEKS 20-44]</li>
      <li>DELIVERABLE: Link with known eQTL. Meynert.[WEEKS 0-40]</li>
      <li>DELIVERABLE: WGS. Baillie.(£606 - 27% unit cost for seq)[WEEKS 20-30]</li>
      <li>DELIVERABLE: Identify high-consequence variants in WGS. Sancho Shimizu.[WEEKS 30-45]</li>
    </ul>
  </li>
</ul>

<p><strong>WP6. Immunology</strong> (<em>Openshaw(lead), Klenerman(deputy), Thwaites, Zambon, Ijaz, Thomson, Turtle, Dunning, Solomon, Stuart</em>)</p>

<ul>
  <li>Describe and characterise the host immune response.(<em>Openshaw</em>)[WEEKS 12-22]
    <ul>
      <li>DELIVERABLE: Cytokine and proteomic analyses in respiratory specimens. Openshaw.[WEEKS 12-30]</li>
      <li>DELIVERABLE: Cytokine analyses in blood. Openshaw.[WEEKS 12-30]</li>
      <li>DELIVERABLE: Proteomic analyses in 60 blood samples. Semple.[WEEKS 12-22]</li>
    </ul>
  </li>
  <li>Determine effects of prior immunity to coronaviruses on susceptibility.(<em>Thomson, Zambon, Ijaz</em>)[WEEKS 0-40]
    <ul>
      <li>DELIVERABLE: Serological tests for prior exposure to coronaviruses. Ijaz.[WEEKS 6-18]</li>
      <li>DELIVERABLE: NGS Serological tests for prior exposure to all resp viruses. Thomson.[WEEKS 12-22]</li>
      <li>DELIVERABLE: Integrate clinical,  transcriptomic,  NGS serology and neutralisation assays to determine ADE effect. Ballie.[WEEKS 30-40]</li>
    </ul>
  </li>
  <li>Describe B&amp;T cell responses in a subset of patients(<em>Klenerman</em>)[WEEKS 4-16]
    <ul>
      <li>DELIVERABLE: Obtain PBMCs from a subset of acute cases. Turtle.[WEEKS 4-16]</li>
      <li>DELIVERABLE: T-cell functional assays. Klenerman.[WEEKS 4-16]</li>
    </ul>
  </li>
  <li>Identify antigens targeted by neutralising antibodies.(<em>Stuart</em>)[WEEKS 0-26]
    <ul>
      <li>DELIVERABLE: Determine structure of antibody-binding sites. Stuart.[WEEKS 0-40]</li>
    </ul>
  </li>
  <li>Identify nasal cell populations supporting viral replication(<em>Thwaites</em>)[WEEKS 8-14]
    <ul>
      <li>DELIVERABLE: Single-cell (nucelar) sequencing on 20 nasopharyngeal (SAM) samples. Thwaites.[WEEKS 8-14]</li>
    </ul>
  </li>
  <li>Identify circulating immune cell subpopulations associated with severe disease.(<em>Klenerman</em>)[WEEKS 0-20]
    <ul>
      <li>DELIVERABLE: Single-cell (nuclear) sequencing on 60 selected frozen cell pellet samples. Klenerman.[WEEKS 8-14]</li>
    </ul>
  </li>
</ul>

<p><strong>WP7. Diagnostics and immunotherapeutics</strong> (<em>Mentzer, Ijaz, Zambon(co-leads), Pollakis, Noursadeghi, Klenerman</em>)</p>

<ul>
  <li>Obtain large-volume convalescent samples from 8 patients.(<em>Mentzer</em>)[WEEKS 0-10]
    <ul>
      <li>DELIVERABLE: Select 8 patients and arrange transfer to Oxford. Mentzer.[WEEKS 0-4]</li>
      <li>DELIVERABLE: Obtain 8x200mls convalescent plasma and PBMCs. Mentzer.[WEEKS 0-4]</li>
    </ul>
  </li>
  <li>Distribute serum and PBMCs to PHE and external groups(<em>Semple, Mentzer</em>)[WEEKS 0-60]
    <ul>
      <li>DELIVERABLE: Cyropreserve and distribute convalescent PMBC as instructed by IDAMAC. Mentzer.[WEEKS 0-10]</li>
      <li>DELIVERABLE: Store and distribute convalescent plasma as instructed by IDAMAC. Mentzer.[WEEKS 0-10]</li>
      <li>DELIVERABLE: Provide samples to define international reference standard. Mentzer.[WEEKS 0-1]</li>
    </ul>
  </li>
  <li>Establish open-access platform for large-scale diagnostic evaluation.(<em>Mentzer</em>)[WEEKS 0-52]
    <ul>
      <li>DELIVERABLE: Platform for comparisons of diagnostics. Mentzer.[WEEKS 8-65]</li>
      <li>DELIVERABLE: Validate oral fluid test for population serology. Tedder.[WEEKS 20-32]</li>
    </ul>
  </li>
  <li>CL2 SARS-CoV-2 virus neutralization assay(<em>Pollakis/Paxton</em>)[WEEKS 0-52]
    <ul>
      <li>DELIVERABLE: Produce retroviral pseudo-virus,  optimize for infectivity. Pollakis.[WEEKS 0-6]</li>
      <li>DELIVERABLE: Determine best target cell for infection and sensitive readout. Pollakis.[WEEKS 4-8]</li>
      <li>DELIVERABLE: Selection of high neutralizing sera for relative quantification of responses. Paxton.[WEEKS 8-20]</li>
      <li>DELIVERABLE: Develop SARS-CoV-2-specific neutralization protocol. Paxton.[WEEKS 8-20]</li>
      <li>DELIVERABLE: Convalescent sera neutralizing response measurements and cross-validation with serology assays. Paxton/Pollakis.[WEEKS 20-52]</li>
      <li>DELIVERABLE: Upscaling of neutralization assay for high throughput. Pollakis.[WEEKS 20-34]</li>
      <li>DELIVERABLE: High throughput neutralization measurement on all samples. Pollakis.[WEEKS 35-48]</li>
    </ul>
  </li>
</ul>

<p><strong>WP8. Facilitating translation to local clinical guidelines</strong> (<em>Alex(lead), Bach(deputy), Grover, Baillie, Russell</em>)</p>

<ul>
  <li>AI-driven guideline/protocol sharing system for local clinicians.(<em>Alex, Bach</em>)[WEEKS 0-30]
    <ul>
      <li>DELIVERABLE: Collate, compare and present local clinical guidelines. Russell.[WEEKS 0-10]</li>
      <li>DELIVERABLE: Create AI engine. Alex.[WEEKS 0-30].</li>
      <li>DELIVERABLE: Create web interface. Bach.[WEEKS 0-20].</li>
    </ul>
  </li>
</ul>

<p><strong>WP9. Long-term follow-up (unfunded)</strong> (<em>Lone(lead), Semple(deputy), Jenkins, Turtle, Docherty, Baillie, Russell, many others…</em>)</p>
<ul>
  <li>Serial serology (Semple, Turtle)
    <ul>
      <li>DELIVERABLE: Serial serology and immunology follow-up of selected cases. Semple.</li>
    </ul>
  </li>
  <li>Clincal follow-up (Lone, Jenkins)
    <ul>
      <li>DELIVERABLE: Data linkage follow-up of all cases. Lone.</li>
      <li>DELIVERABLE: Clinical follow-up of selected cases. Jenkins.</li>
    </ul>
  </li>
</ul>



      </section>
    </div>
    
  </body>
</html>
