<p>The ODAP is guided and overseen by the ODAP Partnership, a collaboration among nationwide research studies, public health agencies, the NHS and clinicians and scientists.</p>
<h2 id="scope">Scope</h2>
<p>The purpose of the ODAP is to facilitate biomedical research to advance understanding of disease caused by emerging or re-emerging pathogens* or other exposures** of public health interest. Research within the ODAP is strictly limited to this purpose.</p>
<p>*Emerging or re-emerging pathogens of public health interest: this term describes new infectious agents, and new, re-emerging or therapy-resistant forms of existing infectious agents.</p>
<p>**Other exposures of public health interest: this term describes new or unexplained poisoning, or exposure to harmful energy sources such as electromagnetic radiation.</p>
<h1 id="what-it-is">What it is</h1>
<ul>
<li>The purpose of the outbreak data analysis platform is to provide an accessible, usable data resource to enable research relevant to COVID-19 and future outbreaks. This will accelerate scientific understanding of new outbreaks for the benefit of patients and the protection of the public.</li>
<li>It will create a UK-wide capability by curating and linking outbreak relevant data from clinical records, research studies and audit data. It brings together key initiatives and leadership across the UK including ISARIC, COG-UK, MRC CLIMB and GenOMICC.</li>
<li>The platform combines a national Trusted Research Environment (TRE) infrastructure collocated with &gt;£100M of world-class computational and data science capacity including the UK National Supercomputer, with a UK-wide governance framework.</li>
</ul>
<h1 id="what-it-isnt">What it isn’t</h1>
<ul>
<li>It is not a replacement for public health activities. The platform will work closely with public health agencies across the 4 nations and with UK HSA, providing research insights and data feeds where useful to augment surveillance capacity, and a tried-and-tested route to engage additional analytic capacity and expertise from the academic sector.</li>
<li>It is not a viral sequence analysis platform. The excellent MRC CLIMB resource (a partner in this programme) provides excellent resources and infrastructure for analysis, presentation and sharing of viral sequences.</li>
<li>It is not a replacement for existing TREs. The analysis platform is a focused capability of the UKSA-accredited TRE hosted by Public Health Scotland, and will provide curated data feeds to TREs across the UK to facilitate and supplement data held elsewhere as part of a UK federated network.</li>
</ul>
<h1 id="design-of-the-platform">Design of the platform</h1>
<h2 id="data-held">Data held</h2>
<p>The platform already contains a unique aggregation of UK sovereign data assets, including the complete data resources of the ISARIC4C/CO-CIN, GenOMICC, PHOSP and UK-CIC studies, together with viral sequence data from COG-UK, and linkage to NHS clinical records and structured clinical audit data. This creates a unique opportunity to combine clinical, biological, genomics and virology research in as secure, openly-accessible framework. Manual curation of these linked datasets, in a single platform, is a key step to maximise data quality and usability.</p>
<h2 id="compute-power">Compute power</h2>
<p>The platform is located at the UK National Supercomputer, and uses a range of capabilities from rom Private Cloud TRE infrastructure to large shared memory systems and massively parallel processing power. There is currently 2.5Pb of storage in use, expandable to many Petabytes as required. GPU servers are in place for massively-parallel applications such as genomics and image analysis</p>
<p>An API structure has been built for real-time data sharing with other trusted research environments across the UK, and to facilitate efficient data pipelines to supply surveillance feeds to public health.</p>
<h2 id="structure">Structure</h2>
<p>There are two routes of access to the analysis platform (<span class="citation" data-cites="fig:structure">@fig:structure</span>):</p>
<ol type="1">
<li>NHS Trusted Research Environment (Safe Haven) for access to personal clinical data and data collected without explicit consent.</li>
<li>Rapid-access flexible compute for access to non-disclosive research data collected with explicit consent.</li>
</ol>
<p>Within both of these environments there are two levels of access, governed by the data contributors:</p>
<ol type="1">
<li>Publishable “open access” data which any user can use and report as they wish, according to data protection and privacy rules;</li>
<li>Embargoed active research data, shared by academic investigators and available for linked analysis but not for publication without agreement from all contributors.</li>
</ol>
<p>This design is intended to demonstrate trust in order to encourage immediate contributions of research data from academic collaborators. It makes data available immediately for discovery, whilst protecting the rights of data creators and contributors.</p>
<figure>
<img src="https://isaric4c.net/img/ap/i4c-analysis-platform-updated.png" id="fig:structure" style="width:90.0%" alt="Structure of the Outbreak Analysis Platform" /><figcaption aria-hidden="true">Structure of the Outbreak Analysis Platform</figcaption>
</figure>
<h1 id="research-outputs">Research outputs</h1>
<p>The proposal builds upon an established track record of impact. By generating, integrating and analysing clinical, biological, genetic and virological data on patients with Covid-19 in UK hospitals, the outbreak analysis platform has facilitated research that:</p>
<ul>
<li>provided essential weekly updates to SAGE that guide the public health response <a href="https://isaric4c.net/reports/">isaric4c.net/reports/</a>,</li>
<li>identified host genetic mechanisms of disease, <span class="citation" data-cites="pairocastineirageneticmechanismscritical2021">[@pairocastineirageneticmechanismscritical2021]</span></li>
<li>quantified the role of age, comorbid illness and obesity in disease severity,<span class="citation" data-cites="dochertyfeatures201332020">[@dochertyfeatures201332020]</span></li>
<li>created the global standard ISARIC4C score for prognostication <a href="isaric4c.net/risk">isaric4c.net/risk</a>,<span class="citation" data-cites="knightriskstratificationpatients2020">[@knightriskstratificationpatients2020]</span></li>
<li>determined the impact of long Covid following hospitalisation<span class="citation" data-cites="groupphysicalcognitivemental2021">[@groupphysicalcognitivemental2021]</span></li>
<li>identified the substantial effect of transmission of Covid-19 within hospitals <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/961210/S1056_Contribution_of_nosocomial_infections_to_the_first_wave.pdf">SPI-M/SAGE report</a>,</li>
<li>provided key evidence underlying the choice of therapeutic agents for clinical trials<span class="citation" data-cites="thwaitesinflammatoryprofilesspectrum2021 pairocastineirageneticmechanismscritical2021">[@thwaitesinflammatoryprofilesspectrum2021; @pairocastineirageneticmechanismscritical2021]</span></li>
<li>provided real world data on vaccine effectiveness and failure (SAGE 87 Egan et al, Egan et al.)</li>
<li>observed data supporting identification of high risk groups for vaccination (highlighted in No10 briefing)</li>
<li>described the complications of Covid-19 in hospitalised patients.<span class="citation" data-cites="drakecharacterisationinhospitalcomplications2021">[@drakecharacterisationinhospitalcomplications2021]</span></li>
</ul>
<h1 id="current-data-content">Current data content</h1>
<p>This platform now serves as a hub for a coordinated UK national research response to COVID-19. Data are included from:</p>
<ul>
<li>ISARIC4C tier 0: (unconsented) prospective clinical data from 298,836 cases</li>
<li>ISARIC4C tiers 1 and 2: serial multiomic assays from research samples of blood, respiratory secretions, urine, and stool from 2,505 cases</li>
<li>COG-UK: (unconsented) summary variant data from COG-UK viral sequencing study is already included for matched patients</li>
<li>GenOMICC study complete data: microarray/whole genome sequence data from 18,158 cases</li>
<li>PHOSP complete data: follow-up clinical and biological data generated by the Post-Hospitalisation for COVID-19 follow-up study (1,075 cases)</li>
<li>UK-CIC: deep immunological phenotyping data from across the UK Coronavirus Immunology Consortium, using ISARIC4C samples and local collections.</li>
</ul>
<p>Research data within the analysis platform is already linked to:</p>
<ul>
<li>NHS Scotland primary, secondary care and death records</li>
<li>NHS Digital health records data</li>
</ul>
<p>In future, plans are in place to transfer data to link with:</p>
<ul>
<li>ICNARC and SICSAG critical care audit databases</li>
<li>NIMS National Immunisation Dataset</li>
<li>Pillar 1 testing</li>
<li>Pillar 2 testing</li>
<li>ONS</li>
</ul>
<h1 id="references">References</h1>
