<!--
Outbreak data analysis platform
-->
<h2 id="data-held">Data held</h2>
<p>The outbreak analysis platform was developed by ISARIC4C to <em>encourgage and facilitate research</em> by collating, linking and presenting data together with high-performance computational capacity.</p>
<p>It provides a clinical and research data integration platform to facilitate integrative analyses of multi-omic, serial disease profiling, stratified by viral variant, clinical phenotype and outcome. The platform is hosted on nationally-leading, exabyte-scale computational infrastructure including state-of-the-art security systems for protection of identifiable data, and high-performance CPU/GPU computing (EPCC, and ARCHER/ARCHER2).</p>
<figure>
<img src="https://isaric4c.net/img/ap/i4c-analysis-platform-updated.png" id="fig:structure" style="width:80.0%" alt="Figure 1: Structure of the ISARIC4C Analysis Platform" /><figcaption aria-hidden="true">Figure 1: Structure of the ISARIC4C Analysis Platform</figcaption>
</figure>
<p>This platform now serves as a hub for a coordinated UK national research response to COVID-19. Data are included from:</p>
<ul>
<li>ISARIC4C tier 0: (unconsented) prospective clinical data from 204489 cases</li>
<li>ISARIC4C tiers 1 and 2: serial multiomic assays from research samples of blood, respiratory secretions, urine, and stool from 2306 cases</li>
<li>COG-UK: (unconsented) summary variant data from COG-UK viral sequencing study is already included for matched patients</li>
<li><a href="https://genomicc.org/uk">GenOMICC study</a> complete data: microarray and whole genome sequence data from 12457 cases</li>
<li>PHOSP complete data: follow-up clinical and biological data generated by the Post-Hospitalisation for COVID-19 follow-up study (1075 cases)</li>
<li>UK-CIC: deep immunological phenotyping data from across the UK Coronavirus Immunology Consortium, using ISARIC4C samples and local collections.</li>
</ul>
<h2 id="linkage-to-clinical-data">Linkage to clinical data</h2>
<p>Research data within the analysis platform is already linked to:</p>
<ul>
<li>NHS Scotland primary, secondary care and death records</li>
<li>NHS Digital health records data</li>
</ul>
<p>In future, plans are in place to transfer data to link with:</p>
<ul>
<li>ICNARC and SICSAG critical care audit databases</li>
<li>NIMS National Immunisation Dataset</li>
<li>Pillar 1 testing</li>
<li>Pillar 2 testing</li>
<li>ONS</li>
</ul>
<h2 id="research-outputs">Research outputs</h2>
<p>The <a href="https://isaric4c.net">ISARIC Coronavirus Clinical Characterisation Consortium (4C)</a> is the largest observational study of hospitalised patients with COVID-19 anyhwere in the world. By generating, integrating and analysing clinical, biological, genetic and virological data on patients with Covid-19 in UK hospitals (Figure 2), ISARIC4C has provided essential weekly updates to SAGE that guide the public health response, and enabled understanding of the clinical features,<span class="citation" data-cites="dochertyFeatures201332020"><sup><a href="#ref-dochertyFeatures201332020" role="doc-biblioref">1</a></sup></span> prognostication,<span class="citation" data-cites="KnightRiskstratificationpatients2020"><sup><a href="#ref-KnightRiskstratificationpatients2020" role="doc-biblioref">2</a></sup></span> disease biology<span class="citation" data-cites="thwaitesElevatedAntiviralMyeloid2020"><sup><a href="#ref-thwaitesElevatedAntiviralMyeloid2020" role="doc-biblioref">3</a></sup></span> and host genetics.<span class="citation" data-cites="PairoCastineiraGeneticmechanismscritical2020"><sup><a href="#ref-PairoCastineiraGeneticmechanismscritical2020" role="doc-biblioref">4</a></sup></span></p>
<h2 id="analysis-platform-structure">Analysis platform structure</h2>
<p>There are two routes of access to the analysis platform (Figure 1): 1. NHS Trusted Research Environment (Safe Haven) for access to personal clinical data and data collected without explicit consent. 2. Rapid-access flexible compute for access to non-disclosive research data collected with explicit consent.</p>
<p>Within both of these environments there is an additional division in the data: 1. Publishable “open access” data which any user can use and report as they wish, according to data protection and privacy rules; 2. Embargoed active research data, shared by academic investigators and available for linked analysis but not for publication without agreement from all contributors.</p>
<p>This design is intended to build trust in order to encourage immediate contributions of research data from academic collaborators.</p>
<figure>
<img src="https://isaric4c.net/img/ap/i4c-map-updated.png" id="fig:map" style="width:60.0%" alt="Figure 2: ISARIC4C study and data analysis platform" /><figcaption aria-hidden="true">Figure 2: ISARIC4C study and data analysis platform</figcaption>
</figure>
<h2 id="future-plans">Future plans</h2>
<p>Rapid addition of viral sequence data from the COG-UK platform will enable real-time detection of the clinical impact of new viral strains, in-depth biological study of reinfection, and host:pathogen interactions at a genetic and mechanistic level.</p>
<!--
# Background


## Earning trust from data contributors

The default position is that data are contributed under embargo, prohibiting publication or general release until authorised by the data contributor. All contributors will agree to abide by this rule in good faith. Embargoed data will be available to other contributors during the embargo period, and will be released into the open analysis platform at or before the time of the first pre-print report.

A critical determinant of success is building sufficient trust among contributors to ensure that data are contributed in an accessible format as early as possible. Data sharing within the ISARIC4C consortium continues to have the support and goodwill of contributors, because:
- there is a palpable urgency created by the COVID-19 crisis;
- the platform has earned the trust of contributors and will maintain it by enforcing embargo rules;
- there is a clear expectation from patients, the public, funders and government;
- there is primary benefit to data contributors to gain access to other unpublished data and analysis platforms.

## Principles

ISARIC4C is built on top of existing pandemic preparedness infrastructure, designed, established, maintained and tested during the interpandemic period (@fig:map),[@dunningOpenSourceClinical2014] and harmonised across the world.[@akhvledianiGlobalOutbreakResearch2020a] It is an open-access national resource: we have already shared data on 204489 participants and 4273 samples with 26 external groups.

The success of ISARIC4C is largely due to the following foundational principles:

- no group, funder, collaborator or other party will have exclusive access to data or samples
- consortium resources (samples, data and funds) will be prioritised according to likelihood of rapid impact on the COVID-19 pandemic
- all data generated using ISARIC4C resources is shared in a machine-readable format within the Integrated Analysis Platform

## Open analysis platform for deidentified data

The analysis platform is being used to provide itegrated analyses of genetic associations with multiple phenotypes,[@canela-xandriAtlasGeneticAssociations2018] functional genomics,[@fantom5_2014] and multi-omics critical illness trajectories,[@neytonMolecularPatternsAcute2020] within the largest clinical study of COVID-19 anywhere in the world.[@DochertyFeatures20133patients2020]

The platform hosts overlapping datasets from across the UK. Individual patient consent enables sharing of linked whole-genome sequence data, whole-blood transcriptomics, proteomics, cytokine measurements, viral load and sequence, and clinical data. This will enable a range of discovery science with direct therapeutic applications, including subphenotype classification and extended causal inference using Mendelian randomisation and related approaches.

Providing clean, linked, deidentified data in a format that is easily accessible to researchers from a range of backgrounds requires staff with a high level of skill in clinical epidemiology, data science, and software engineering. Data will be systematically cleansed and linked, missing data completed in an iterative process interacting with analysis teams, and presented in curated flat files and through an integrated relational database. This will be presented to users through four interfaces:

1. a user-friendly browsable interface enabling data selection and subgrouping through dropdown menus to subset patient populations by clinical and biological data and run *de novo* GWAS analyses using a GPU platform (GOLEM, Tenesa group), multivariable regression, propensity-matching, unsupervised clustering and other analyses.

2. flexible analysis through bespoke, secure virtual machines operated through a command line interface providing access to R, Python, and other software as required by the user.

3. a well-documented application programming interface (API) enabling external computational queries. This allows all data in the ISARIC4C platform to contribute to federated data analysis frameworks at national and international level. Collaborating groups such as OPENSafely and Genomics England will be able to run queries seamlessly from external platforms.

4. a limited, anonymised, downloadable dataset comprising key variables from all participants.

Deidentified data will be available openly to *bona fide* researchers for unrestricted analyses

## Data safe haven

A linked, secure NHS data safe haven will provide access to identifiable data, and data collected without individual patient consent, for qualified, approved researchers performing research to improve patient care. This incorporates full ISARIC COVID case report forms for 46,000 patients, together with health records linkage (CAG section 251 and PBPP approvals in place).

This will enable detailed, rich clinical analyses with corrections for confounding and bias caused by social factors, comorbid illness and medications, and opens a range of detailed information to characterise acute disease using clinical measurements acquired from electronic health records.


| Dataset                           | Governance responsibility | Delegates  |
| --------------------------------- | ------------------------ | -------------------------------------------- |
| [ISARIC4C TIERS 0(CO-CIN)/1/2](https://isaric4c.net/analysis-platform)   | Kenneth Baillie (for IDAMAC) | Calum Semple; Gary Leeming; Andy Law;  Wilna Oosthuyzen |
| COG-UK | Sharon Peacock   | Ewan Harrison |
| PHOSP  | Chris Brightling | Aarti Parmar  |
| [GenOMICC](https://genomicc.org/data)   | Kenneth Baillie  | Andy Law; Alison Meynert; Wilna Oosthuyzen  |
| UK CIC | Paul Moss  |   |
| [SICSAG](https://www.isdscotland.org/Health-Topics/Scottish-Healthcare-Audits/Scottish-Intensive-Care-Society-Audit-Group/) | Naz Lone (for SICSAG steering committee) |   |
| [Outpatients (SMR00)](https://www.ndc.scot.nhs.uk/National-Datasets/data.asp?ID=1&SubID=4) |   |   |
| [GeneralAcute and Inpatient Day Case dataset (SMR01)](https://www.ndc.scot.nhs.uk/National-Datasets/data.asp?ID=1&SubID=5)  |   |   |
| [Prescribing Information System (PIS)](https://www.isdscotland.org/health-topics/prescribing-and-medicines/_docs/Open_Data_Glossary_of_Terms.pdf?1) |   |   |
| [NRS Deaths](https://www.ndc.scot.nhs.uk/National-Datasets/data.asp?ID=3&SubID=13)   |   |   |
| Primary Care | GP committee  |   |
| NHS England  |   |   |
| NIMS National Immunisation Dataset   |   |   |
| Pillar 1 testing   |   |   |
| Pillar 2 testing   |   |   |

-->
<!--
## Use cases

Genomics person docker containers

-->
<h1 class="unnumbered" id="references">References</h1>
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-dochertyFeatures201332020" class="csl-entry" role="doc-biblioentry">
1. Docherty, A.B., Harrison, E.M., Green, C.A., Hardwick, H.E., Pius, R., Norman, L., Holden, K.A., Read, J.M., Dondelinger, F., Carson, G., Merson, L., Lee, J., Plotkin, D., Sigfrid, L., Halpin, S., Jackson, C., Gamble, C., Horby, P.W., Nguyen-Van-Tam, J.S., Ho, A., Russell, C.D., Dunning, J., Openshaw, P.J., Baillie, J.K. &amp; Semple, M.G. Features of 20 133 <span>UK</span> patients in hospital with covid-19 using the <span>ISARIC WHO Clinical Characterisation Protocol</span>: Prospective observational cohort study. <em>BMJ</em> <strong>369</strong>, (2020).
</div>
<div id="ref-KnightRiskstratificationpatients2020" class="csl-entry" role="doc-biblioentry">
2. Knight, S.R., Ho, A., Pius, R., Buchan, I., Carson, G., Drake, T.M., Dunning, J., Fairfield, C.J., Gamble, C., Green, C.A., Gupta, R., Halpin, S., Hardwick, H.E., Holden, K.A., Horby, P.W., Jackson, C., Mclean, K.A., Merson, L., Nguyen-Van-Tam, J.S., Norman, L., Noursadeghi, M., Olliaro, P.L., Pritchard, M.G., Russell, C.D., Shaw, C.A., Sheikh, A., Solomon, T., Sudlow, C., Swann, O.V., Turtle, L.C., Openshaw, P.J., Baillie, J.K., Semple, M.G., Docherty, A.B. &amp; Harrison, E.M. Risk stratification of patients admitted to hospital with covid-19 using the ISARIC WHO clinical characterisation protocol: Development and validation of the 4C mortality score. <em>BMJ (Clinical research ed.)</em> <strong>370</strong>, m3339(2020).
</div>
<div id="ref-thwaitesElevatedAntiviralMyeloid2020" class="csl-entry" role="doc-biblioentry">
3. Thwaites, R., Uruchurtu, A.S.S., Siggins, M., Liew, F., Russell, C.D., Moore, S., Carter, E., Abrams, S., Short, C.-E., Thaventhiran, T., Bergstrom, E., Gardener, Z., Ascough, S., Chiu, C., Docherty, A.B., Hunt, D., Crow, Y., Solomon, T., Taylor, G., Turtle, L., Harrison, E.M., Semple, M.G., Baillie, J.K. &amp; Openshaw, P.J. Elevated antiviral, myeloid and endothelial inflammatory markers in severe <span>COVID</span>-19. <em>medRxiv</em> 2020.10.08.20209411(2020).doi:<a href="https://doi.org/10.1101/2020.10.08.20209411">10.1101/2020.10.08.20209411</a>
</div>
<div id="ref-PairoCastineiraGeneticmechanismscritical2020" class="csl-entry" role="doc-biblioentry">
4. Pairo-Castineira, E., Clohisey, S., Klaric, L., Bretherick, A.D., Rawlik, K., Pasko, D., Walker, S., Parkinson, N., Fourman, M.H., Russell, C.D., Furniss, J., Richmond, A., Gountouna, E., Wrobel, N., Harrison, D., Wang, B., Wu, Y., Meynert, A., Griffiths, F., Oosthuyzen, W., Kousathanas, A., Moutsianas, L., Yang, Z., Zhai, R., Zheng, C., Grimes, G., Beale, R., Millar, J., Shih, B., Keating, S., Zechner, M., Haley, C., Porteous, D.J., Hayward, C., Yang, J., Knight, J., Summers, C., Shankar-Hari, M., Klenerman, P., Turtle, L., Ho, A., Moore, S.C., Hinds, C., Horby, P., Nichol, A., Maslove, D., Ling, L., McAuley, D., Montgomery, H., Walsh, T., Pereira, A., Renieri, A., Shen, X., Ponting, C.P., Fawkes, A., Tenesa, A., Caulfield, M., Scott, R., Rowan, K., Murphy, L., Openshaw, P.J.M., Semple, M.G., Law, A., Vitart, V., Wilson, J.F. &amp; Baillie, J.K. Genetic mechanisms of critical illness in covid-19. <em>Nature</em> (2020).doi:<a href="https://doi.org/10.1038/s41586-020-03065-y">10.1038/s41586-020-03065-y</a>
</div>
</div>
